contracts/libraries/HyperdriveExecution.sol

```diff
@@ -86,114 +86,6 @@ library HyperdriveExecutionLibrary {
         );
     }
 
-    /// @dev Calculates the result of opening a long with hyperdrive.
-    /// @param _asBase Whether to use hyperdrive's base asset.
-    /// @param _amount Amount of assets to spend.
-    /// @return Amount of bonds received.
-    function previewOpenLong(
```

Comment:
Question: Why did we get rid of this and `_calculateOpenLong`?

Reply:
Were they not used before, or did you just delete the uses? What I'm trying to understand is how we were using this before (maybe the answer is that we weren't) and why is it okay to stop using it now.

Reply:
They weren't used before.... they were originally included when I was misguided about slippage on `openLong`, but after we discussed how we don't need to account for slippage due to atomicity, they were removed.

Reply:
Gotcha, thanks for the context.